sshlib/src/main/java/com/trilead/ssh2/auth/SignatureProxy.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017 Jonas Dippel, Michael Perk, Marc Totzke
+ */
+
+package com.trilead.ssh2.auth;
+
+import java.io.IOException;
+import java.security.PublicKey;
+
+import jdk.nashorn.internal.objects.annotations.Getter;
```

Comment:
This import as well as the `@Getter` line below caused problems during compilation for me (and looks like it doesn't belong here - Nashorn is a Javascript engine, isn't it!?)
After removing them, everything worked fine.

Reply:
Thanks, I somehow overlooked that.
It's just an annotation and it should not cause problems with a recent enough JDK >= 8.
But looking at the definition of jdk.nashorn.internal.objects.annotations.Getter, it's not a general 'Getter' annotation but specific to a JS data property getter. I removed it.